packages/expo-notifications/src/NotificationCategoriesModule.ts

```diff
@@ -1,8 +1,8 @@
 import { NotificationAction } from './Notifications.types';
 
 export default {
-  async getNotificationCategoriesAsync(): Promise<null> {
-    return null;
+  async getNotificationCategoriesAsync(): Promise<never[]> {
```

Comment:
```suggestion
  async getNotificationCategoriesAsync(): Promise<NotificationCategory[]> {
```

(so that the types of the default import/export are correct in apps)